source/utilities.tex

```diff
@@ -4724,7 +4724,8 @@
 
 \pnum
 \begin{note}
-A \tcode{variant} might not hold a value if an exception is thrown during a
+It is possible for a \tcode{variant} not to hold a value
+if an exception is thrown during a
```

Comment:
Another awkward "possible not to ..." but I don't have a better suggestion.

Reply:
Ah , in [variant.variant.general] we already say "or holds no value", so if we don't mind using "hold no value" as the opposite of "hold a value", we could say "It is possible for a `variant` to hold no value if ...".

Reply:
fixed